### PR TITLE
troubleshooting tc workers

### PR DESCRIPTION
**Pull request by escapewindow**

### CODE REVIEW

File: taskcluster/troubleshooting_workers.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Worker Manager errors
+---------------------
+
+The `worker manager UI <https://firefox-ci-tc.services.mozilla.com/worker-manager>`__ lists the various pools. It shows current capacity and pending tasks, as well as links to view the workers and recent worker manager error messages, for each pool.
+
+Drilling down into each workerType should show you the config. Generally this will be an expanded version of the `ci-configuration worker-pools config <https://hg.mozilla.org/ci/ci-configuration/file/tip/worker-pools.yml>`__.
+
+Common types of error messages
+------------------------------
```
Comment (MihaiTabara) [inline taskcluster/troubleshooting_workers.rst:23]: Follow-up, not blocking, we should probably enumerate these at some point, if more are expected to be added. But for now it looks good.

**Review by MihaiTabara — APPROVED** [review]
This is treasure, thanks a lot for adding these Aki!

**Review by JohanLorenzo — APPROVED** [review]
I agree with Mihai! Thank you for writing this down, Aki. This matches what I've been doing so far, and that knowledge was technically just in our head 😁

File: taskcluster/troubleshooting_workers.rst (hunk 1 above)
Comment (escapewindow) [reply taskcluster/troubleshooting_workers.rst:23]: +1. That's why I added `(This page needs fleshing out. Ideally we have a runbook where we can go from symptom to solution.)` to the top :)

**Pull request merged**